### PR TITLE
Backend: setup ESLint+ lint all JS files

### PR DESCRIPTION
**Pull request by cawel**
## Done in this PR

In the backend:
* installed ESLint (enabling all [recommended rules](https://eslint.org/docs/rules/))
* added a `npm run lint` npm task in `package.json` to run ESLint on demand
* linted all JS files in the backend (hence why this PR touched so many files)

## Suggestions

* I would invite you to install `Prettier` in your IDE. That way, each time you save a file, it could format it using the `.eslintrc.json` file located in the backend project root.
* adding a pre-push git hook which would run `npm run lint` could be a way to enforce ESLint rules and make sure there are no JS syntax errors entering the `master` branch.
* feel free to add/change the ESLint rules, if you have strong preferences (e.g. the single/double quotes rule is often a controversial one)
* I would invite you to setup ESLint for the frontend project as well

### CODE REVIEW

File: back-end/db/seeds/01_users.sql [hunk 1]
```diff
@@ -1,34 +1,34 @@
 INSERT INTO users (username, first_name, last_name, email, password, avatar, adventurer)
 VALUES
   (
-    'BobRobertson', 'Bob', 'Robertson', 'bob@example.com', '$2b$10$xPttDUv.c13m9X1ni9CqEOFk1P5exXZeq.2LL.YrztVIWMxi4FTVm', '/images/defaultAvatar.png', true
```
Comment (FelixPriori) [inline back-end/db/seeds/01_users.sql:4]: The link has since been updated. :)

File: client/src/components/Profile/Profile.js [hunk 2]
```diff
@@ -5,13 +5,13 @@ import BadgeBox from '../BadgeBox/BadgeBox';
...
 export default function Profile(props) {
 
-  const { username, first_name, last_name, email, adventurer, avatar } = props.state.userData;
+  const { username, first_name, last_name, email, adventurer } = props.state.userData;
   const { userBadges } = props.state;
 
   return (
     <section className='profile'>
       <h2>Your profile</h2>
-      <img alt="avatar" src={avatar} />
```
Comment (FelixPriori) [inline client/src/components/Profile/Profile.js:14]: This should also be in there :)

Comment (cawel) [reply client/src/components/Profile/Profile.js:14]: Sorry about that. Should be fixed now!

File: back-end/db/seeds/01_users.sql (hunk 1 above)
Comment (cawel) [reply back-end/db/seeds/01_users.sql]: Sorry about that. Should be fixed now!

**Pull request merged**